examples/truncate-text-bento.amp.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
```

Comment:
We have this less-than-great title all around.

```suggestion
  <title>amp-truncate-text</title>
```